### PR TITLE
IDEX-2578: Developed machine perspective view.

### PR DESCRIPTION
**Pull request by dimasnurenko**
@vparfonov @azatsarynnyy

### CODE REVIEW

File: plugin-machine/che-plugin-machine-ext-machine/src/main/java/org/eclipse/che/ide/extension/machine/client/machine/Machine.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * The class which describes machine entity. The class is wrapper of MachineDescriptor.
+ *
+ * @author Dmitry Shnurenko
+ */
+public class Machine {
+
+    private MachineDescriptor descriptor;
+
+    @Inject
+    public Machine() {
```
Comment (vparfonov) [inline plugin-machine/che-plugin-machine-ext-machine/src/main/java/org/eclipse/che/ide/extension/machine/client/machine/Machine.java]: looks like not need

File: plugin-machine/che-plugin-machine-ext-machine/src/main/java/org/eclipse/che/ide/extension/machine/client/perspective/widgets/machine/info/InfoContainerPresenter.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+/**
+ * The class is a container for tab panels which display additional information about machine and adds ability to control machine's
+ * processes. The class is a wrapper of {@link TabContainerPresenter} to use logic  of {@link TabContainerPresenter} for control tabs
+ * instead {@link PartStackPresenter}
+ *
+ * @author Dmitry Shnurenko
+ */
+@Singleton
+public class InfoContainerPresenter extends PartStackPresenter {
```
Comment (vparfonov) [inline plugin-machine/che-plugin-machine-ext-machine/src/main/java/org/eclipse/che/ide/extension/machine/client/perspective/widgets/machine/info/InfoContainerPresenter.java]: not good class name

File: plugin-machine/che-plugin-machine-ext-machine/pom.xml [hunk 3]
```diff
@@ -123,7 +123,7 @@
     </dependencies>
     <build>
         <sourceDirectory>src/main/java</sourceDirectory>
-        <testSourceDirectory>src/test/java</testSourceDirectory>
```
Comment (vparfonov) [inline plugin-machine/che-plugin-machine-ext-machine/pom.xml]: must be src/test/java

**Comment by vparfonov** [comment]
fix comment and wil be ok

File: plugin-machine/che-plugin-machine-ext-machine/src/main/java/org/eclipse/che/ide/extension/machine/client/perspective/widgets/machine/panel/MachinePanelPresenter.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        this.view = view;
+        this.service = service;
+        this.entityFactory = entityFactory;
+        this.widgetsFactory = widgetsFactory;
+        this.locale = locale;
+        this.infoContainer = infoContainer;
+    }
+
+    /** Gets all machines and adds them to special place on view. */
+    public void getMachines() {
```
Comment (sleshchenko) [inline plugin-machine/che-plugin-machine-ext-machine/src/main/java/org/eclipse/che/ide/extension/machine/client/perspective/widgets/machine/panel/MachinePanelPresenter.java]: @dimasnurenko Method with name that start with "get" and has return type void looks strange. Maybe make sense think about another method's name.

**Pull request merged**